model.py

```diff
@@ -55,9 +74,9 @@ class Setting(db.Model):
 
     __tablename__ = "settings"
 
-    setting_id = db.Column(db.Integer, autoincrement=True, primary_key=True)
-    setting_name = db.Column(db.String(100), unique=True)
-    setting_default_value = db.Column(db.String(100))
+    id = db.Column(db.Integer, primary_key=True)
+    name = db.Column(db.String(100), unique=True)
```

Comment:
Again, `setting.setting_name` is redundant. 

Simply `setting.name` is enough and this goes for all model attributes.